pages.it/common/opusenc.md

```diff
@@ -0,0 +1,20 @@
+# opusenc
+
+> Converte audio WAV o FLAC in Opus.
+> Maggiori informazioni: <https://opus-codec.org/docs/opus-tools/opusenc.html>.
+
+- Converte un file WAV in un file Opus usando le opzioni predefinite:
+
+`opusenc {{percorso/all/input.wav}} {{percorso/all/output}}.opus`
```

Comment:
You have `input` and `output` here.

Reply:
I know. Still, you sometimes say `input` or `output` even in Italian in some case.
Some possible translations might be `file_di_input.wav`/`file_di_output.opus` or `file_originale.wav`/`file_modificato.opus`. I wonder which one is the best.
I see there is an Italian maintainer (@mebeim ), maybe he can give his opinion.

Reply:
Also @glenacota

Reply:
I think input and output are pretty safe to use also in Italian. But I prefer your other proposal `file_originale.wav` and `file_modificato.opus`, though :)